tutorials/plot_sensors_time_frequency.py

```diff
@@ -70,12 +72,103 @@
 
 ax.plot(freqs, psds_mean, color='k')
 ax.fill_between(freqs, psds_mean - psds_std, psds_mean + psds_std,
-                color='k', alpha=.5)
+                color='k', alpha=.15, lw=0)
 ax.set(title='Multitaper PSD (gradiometers)', xlabel='Frequency',
        ylabel='Power Spectral Density (dB)')
 plt.show()
 
 ###############################################################################
+# To see the difference between :func:`psd_multitaper` and :func:`psd_welch`
+# - we are going to compare PSDs obtained with each function.
+ch_index = epochs.ch_names.index('MEG 2333')
```

Comment:
what makes sense to me if to show how a trimmed/robust mean works better than a mean for example if you don't discard artifacts in the data. Basically illustrate the benefit of the feature on a problem that makes sense. for me the epochs with reject are already pretty clean so I don't expect a strong for your new feature there. wdyt?